packages/redux-api-auth-middleware/src/store/reducer.js

```diff
@@ -0,0 +1,42 @@
+import { calculateJWTTokenExpirationDate } from '../helpers';
+import { CLEAR_TOKEN, REFRESH_TOKEN_FAILURE, REFRESH_TOKEN_SUCCESS, SET_TOKEN } from './types';
+import type { GetExpirationTimestamp, SetTokenAction, State } from '../types';
+
+export const initialState: State = {
+  authToken: null,
+  refreshToken: null,
+  expires: 0,
+};
+
+export default function({
+  getExpirationTimestamp = calculateJWTTokenExpirationDate,
+}: GetExpirationTimestamp<number> = {}) {
+  return function authReducer(
+    state: State = initialState,
+    action: SetTokenAction,
+    tokenExpiration: GetExpirationTimestamp<number> = getExpirationTimestamp
```

Comment:
I think this is a leftover from refactoring? Reducers accept only 2 params and `getExpirationTimestamp` is already specified.